x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/LogicalPlanOptimizerTests.java

```diff
@@ -211,6 +211,10 @@ public class LogicalPlanOptimizerTests extends AbstractLogicalPlanOptimizerTests
     private static final LiteralsOnTheRight LITERALS_ON_THE_RIGHT = new LiteralsOnTheRight();
 
     public void testEvalWithScoreImplicitLimit() {
+        assumeTrue(
```

Comment:
We can just check the capability.
As an example:

https://github.com/elastic/elasticsearch/blob/a654efe6bf4139149af74c9b09260e50a6f4a283/x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/VerifierTests.java#L2031

but the capability we want to check here is SCORE_FUNCTION from:
https://github.com/elastic/elasticsearch/blob/6beabf4d4bec05486cae1cec950e22e918b4a278/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java#L1121